src/methods/birth-date.js

```diff
@@ -25,8 +25,12 @@ const birthDate = customer => {
  * @returns {string}
  *
  * @example
- * // TODO
- *
+ * costumer will be the parameter of this function and as result will return his birth date
+ * var customer = object (body customer)
```

Comment:
this code isn't valid, i think you should remove it, customer is properly declared as const below